app/views/ThreadMessagesView/Dropdown/DropdownItemHeader.tsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import { FILTER } from '../filters';
 import I18n from '../../../i18n';
 import DropdownItem from './DropdownItem';
 
-const DropdownItemHeader = ({ currentFilter, onPress }) => {
+interface IDropdownItemHeader {
+	currentFilter: string;
```

Comment:
You could type the way you did here to https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3538/files#diff-0522989489ae1ca9184bd55e549861fc816ac4e054faf5210ab3ad7bfd60feaaR93